libraries/apollo-tooling/src/main/kotlin/com/apollographql/apollo3/tooling/SchemaDownloader.kt

```diff
@@ -185,8 +182,23 @@ object SchemaDownloader {
 
   inline fun <reified T> Any?.cast() = this as? T
 
-  private fun getIntrospectionQuery(includeDeprecatedInputFieldsAndArguments: Boolean): String {
-    val includeDeprecated = if (includeDeprecatedInputFieldsAndArguments) "(includeDeprecated: true)" else ""
+  fun getIntrospectionQuery(specVersion: SpecVersion): String {
```

Comment:
Some unrelated commit got in the way

Reply:
Good catch. I just rebased again. Should be good this time.